.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -300,36 +300,28 @@ jobs:
           awk '/^Failed Tests|Unexpectedly Passed Tests|Unresolved tests|Testing Time/{flag=1}/FAILED: CMakeFiles/{flag=0}flag' e2e.log >> $GITHUB_STEP_SUMMARY
         fi
         exit $exit_code
-    - name: List excluded SYCL CTS categories
-      if: inputs.tests_selector == 'cts' && env.CTS_TESTS_TO_BUILD == ''
-      run: |
-        if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_OCL_CPU
-        fi
-        if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_L0_GPU
-        fi
     - name: Build SYCL CTS tests
       if: inputs.tests_selector == 'cts'
       env:
         CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
       run: |
-        # Do not exclude any category if passing the CTS_TESTS_TO_BUILD variable.
         cts_exclude_filter=""
+        # If CTS_TESTS_TO_BUILD is null - use filter
         if [ -z "$CTS_TESTS_TO_BUILD" ]; then
           if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
             cts_exclude_filter=$PWD/devops/cts_exclude_filter_OCL_CPU
-          fi
-          if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
+          elif [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
             cts_exclude_filter=$PWD/devops/cts_exclude_filter_L0_GPU
           fi
+
+          # List excluded SYCL CTS categories
+          cat $cts_exclude_filter
```

Comment:
Please surround with some extra text explaining what the output is. Grouping like on line 364.

Reply:
Added [783aac9](https://github.com/intel/llvm/pull/14807/commits/783aac9b5bd310312b6e9ec9804af34c7f09f9cb)
https://github.com/intel/llvm/actions/runs/10179152431/job/28154280857